### PR TITLE
[Skill] Add tilelang-op-test-design skill for operator UT test design

### PR DESCRIPTION
**Pull request by bingmeiyou**
## Overview

Provides systematic UT test design methodology for TileLang-Ascend operator development, guiding how to scientifically design test cases based on operator characteristics.

---

### Core Methodology
- **Operator-Feature-Driven**: Classify operators based on three dimensions - hardware characteristics (Cube/Vector), computation steps (Single/Multi/Fusion), and mathematical formulas
- **Four-Layer Test Hierarchy**: L0 (threshold ≤50) → L1 (functional 100-200) → L2 (exception ≤20) → Boundary (edge cases ≤10)
- **Irregular Shape Natural Inclusion**: Tail-block scenarios naturally included in L1, not specially emphasized

### Supported Scenarios
1. Design test configuration from `design.md`
2. Supplement tests from `examples/{op}/*.py`
3. Generate test template from user verbal description
4. Test coverage analysis

---

## File Structure

```
.agents/skills/tilelang-op-test-design/
├── SKILL.md                    # Main document (methodology + test structure example)
└── references/
    ├── operator-category.md    # Operator classification criteria
    └── precision-standard.md   # Precision standard system (rtol/atol per dtype/operator category)
```

---

## Applicable Scenarios

Trigger keywords: design operator test, generate test cases, supplement tests, insufficient test coverage

### CODE REVIEW

**Comment by bingmeiyou** [comment]
re-test

**Comment by z00520135** [comment]
re-build

**Review by benyang0506 — DISMISSED** [review]
lgtm

File: .agents/skills/tilelang-op-test-design/SKILL.md [hunk 1]
```diff
@@ -0,0 +1,613 @@
...
+
+### 2.2 计算类型划分（硬件特性）
+
+**依据**：算子主要使用哪类硬件单元
+
+| 计算类型 | 使用的硬件单元 | 数学公式特征 | 测试重点 |
+|---------|--------------|-------------|---------|
+| **纯 Cube** | Cube 核（矩阵乘单元） | 仅含 matmul / @ | 矩阵维度组合、block size |
+| **纯 Vector** | Vector 核（向量单元） | 无 matmul，仅 element-wise/reduction | dtype 组合、shape 组合 |
+| **混合（CV 融合）** | Cube + Vector 核 | matmul + element-wise 后处理 | workspace 配置、核间协作 |
```
Comment (fuhouyu-hw) [inline .agents/skills/tilelang-op-test-design/SKILL.md]: 如果是A5，后续要考虑新通路

**Review by fuhouyu-hw — APPROVED** [review]
/lgtm

**Review by benyang0506 — APPROVED** [review]
approve
